.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
add the .idea folder to your git ignore, we don't want to source control it.

Reply:
Tanks for your guidance，I'll correct it right away